### PR TITLE
Supporter plus nudge annual copy minimum amounts revised per region

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?
Due to changes in pricing requirements, we need to change the nudge pricing for all regions. The new prices are as shown below ->
UK
Control: 10
VariantA: 25
VariantB:50
US
Control: 10
VariantA: 25
VariantB:50
EU
Control: 10
VariantA: 30
VariantB:50
AU
Control: 10
VariantA: 40
VariantB: 75
INT
Control: 10
VariantA: 30
VariantB: 50
CA
Control: 10
VariantA: 35
VariantB: 60
NZ
Control: 10
VariantA: 40
VariantB: 75

[**Trello Card**](https://trello.com/c/Gd9Ff20Z/1405-increase-nudge-to-recurring-change-the-nudge-pricing-for-regions)

## Is this an AB test?
- [x] Yes
- [ ] No

https://support.thegulocal.com/uk/contribute#ab-nudgeMinAmountsTest=control
https://support.thegulocal.com/uk/contribute#ab-nudgeMinAmountsTest=variantA
https://support.thegulocal.com/uk/contribute#ab-nudgeMinAmountsTest=variantB

## Accessibility test checklist
 - [Tested with screen reader](https://webaim.org/articles/voiceover/)
 - [Navigable with keyboard](https://www.accessibility-developer-guide.com/knowledge/keyboard-only/browsing-websites/)

### CODE REVIEW

**Comment by paul-daniel-dempsey** [comment]
[Increase nudge to recurring: Change the nudge pricing for regions](https://trello.com/c/Gd9Ff20Z/1405-increase-nudge-to-recurring-change-the-nudge-pricing-for-regions)

File: support-frontend/assets/helpers/contributions.ts [hunk 1]
```diff
@@ -282,9 +282,33 @@ function getConfigAbTestMin(
...
+				return 35;
+			}
+			if (
+				countryGroupId === 'EURCountries' ||
+				countryGroupId === 'International'
+			) {
+				return 30;
+			} else {
+				return 25;
+			}
```
Comment (undergroundquizscene) [inline support-frontend/assets/helpers/contributions.ts:300]: Would it be clearer to write this as a switch? I think the following should work:

```suggestion
			switch (countryGroupId) {
				case 'AUDCountries':
				case 'NZDCountries':
					return 40;
				case 'Canada':
					return 35;
				case 'EURCountries':
				case 'International':
					return 30;
				case 'GBPCountries':
				case 'UnitedStates':
					return 25;
			}
```

That way there’s less repetition of the variable name, and all cases are named (don’t have to work out that the default case is UK/US). I think the exhaustiveness checking should still be able to tell that we don’t need a default case.

Comment (paul-daniel-dempsey) [reply support-frontend/assets/helpers/contributions.ts:300]: Yes, sorry had not finished, thought would send out conversions request in a bit on the chat and this wouldnt come up on peoples radar till then.  Will refine now, got to rebase with checkout minimum fix as well first.

File: support-frontend/assets/helpers/contributions.ts [hunk 2]
```diff
@@ -282,9 +282,33 @@ function getConfigAbTestMin(
...
+			if (
+				countryGroupId === 'AUDCountries' ||
+				countryGroupId === 'NZDCountries'
+			) {
+				return 75;
+			} else if (countryGroupId === 'Canada') {
+				return 60;
+			} else {
+				return 50;
+			}
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/helpers/contributions.ts:311]: Yes, sorry had not finished, thought would send out conversions request in a bit on the chat and this wouldnt come up on peoples radar till then.  Will refine now, got to rebase with checkout minimum fix as well first.

File: support-frontend/assets/helpers/contributions.ts (hunk 1 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/helpers/contributions.ts]: done fix

File: support-frontend/assets/helpers/contributions.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/helpers/contributions.ts]: done code fix

File: support-frontend/assets/helpers/contributions.ts (hunk 1 above)
Comment (undergroundquizscene) [reply support-frontend/assets/helpers/contributions.ts]: Ah sorry! I didn’t realise it wasn’t ready for review yet

**Pull request merged**